Dockerfile

```
@@ -1,14 +1,32 @@
+FROM ubuntu:24.04 AS chromaprint-build
+
+RUN apt-get update && \
+    apt-get install -y --no-install-recommends cmake make gcc g++ curl ca-certificates unzip && \
+    apt-get clean && \
+    rm -rf /var/lib/apt/lists/*
+
+WORKDIR /opt/chromaprint
+
+RUN curl -L https://github.com/acoustid/chromaprint/archive/41a3e8fb3eb907d7a0338ada291982672a2226df.zip -o 41a3e8fb3eb907d7a0338ada291982672a2226df.zip && \
```

Hadolint (2.12.0)

[warning] Use WORKDIR to switch to a directory

(DL3003)